src/components/app/data/hooks/useEnterpriseCustomer.js

```diff
@@ -9,14 +9,15 @@ export default function useEnterpriseCustomer(queryOptions = {}) {
   const { select, ...queryOptionsRest } = queryOptions;
   return useEnterpriseLearner({
     ...queryOptionsRest,
-    select: (enterpriseLearner) => {
+    select: (data) => {
+      const transformedData = data.transformed.enterpriseCustomer;
       if (select) {
         return select({
-          original: enterpriseLearner,
-          transformed: enterpriseLearner.enterpriseCustomer,
+          original: data,
```

Comment:
nit: hmm, considering whether we should pass `data` or  `data.original` since the original data is _technically_ `data.original` 🤔